cli/running/process_controller.go

```diff
@@ -111,3 +111,38 @@ func (pc *processController) Stop(waitTimeout time.Duration) error {
 		return err
 	}
 }
+
+// Quit terminates the process.
+func (pc *processController) Quit(waitTimeout time.Duration) error {
```

Comment:
Looks like this function is mostly a duplicate of `Stop`, but with another signal. Please remove code duplication.